core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-layout/edit-ema-layout.component.spec.ts

```diff
@@ -166,12 +170,18 @@ describe('EditEmaLayoutComponent', () => {
         });
 
         it('should trigger a save after 5 secs', fakeAsync(() => {
-            const layoutServiceSave = jest.spyOn(layoutService, 'save');
+            const layoutServiceSave = jest
+                .spyOn(layoutService, 'save')
+                .mockReturnValue(of(PAGE_RESPONSE as unknown as DotPageRender));
```

Comment:
@rjvelazco Is it possible to use the `mockProvider` function of spectator to avoid manually creating spies in each test?

Reply:
Done!